include/openssl/bn.h

```diff
@@ -154,9 +154,10 @@ void BN_CTX_start(BN_CTX *ctx);
 BIGNUM *BN_CTX_get(BN_CTX *ctx);
 void BN_CTX_end(BN_CTX *ctx);
 int BN_rand(BIGNUM *rnd, int bits, int top, int bottom);
-int BN_pseudo_rand(BIGNUM *rnd, int bits, int top, int bottom);
 int BN_rand_range(BIGNUM *rnd, const BIGNUM *range);
-int BN_pseudo_rand_range(BIGNUM *rnd, const BIGNUM *range);
+DEPRECATEDIN_1_1_0(int BN_pseudo_rand(BIGNUM *rnd,
+                                      int bits, int top, int bottom))
+DEPRECATEDIN_1_1_0(int BN_pseudo_rand_range(BIGNUM *rnd, const BIGNUM *range))
```

Comment:
Not ok, this is a change of an existing API.  You may, however, use `DEPRECATEDIN_1_2_0`, it just needs to be defined in `include/openssl/opensslconf.h.in` (I'm surprised, I thought that was already added...  by @mattcaswell if I remember correctly)